### ISSUE TITLE
Epoch 0 lightning_checkpoints are loaded as opposed to latest weights 

### ISSUE THREAD

**Issue by pwang724** [jbohnslav/deepethogram#86]
When training the feature extractor using either the GUI or in CLI ('flow_generator.weights=latest '), I noticed that log outputs

"loading specified weights: ..\models\211216_052127_flow_generator_train\lightning_checkpoints\epoch=0-step=437.ckpt"

Isn't this the checkpoint that is saved after the end of the first epoch, and shouldn't the checkpoint that should be loaded for additional training be "last.ckpt"?

Manual deletion of this old ckpt file solves this issue, but if I'm not mistaken, this should be fixed as forgetting to manually delete the file would render hours and hours of training obsolete...

**Comment by jbohnslav** [comment]
Hi there, 

This is the default behavior. It seems like your best epoch, according to deepethogram's metrics, was the first epoch. It seems like you have some overfitting, so I'd try expanding the size of your dataset or increasing regularization (dropout or l2_sp alpha).
